pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -452,6 +452,17 @@ def test_get_loc_reasonable_key_error(self):
         with pytest.raises(KeyError, match="2000"):
             index.get_loc("1/1/2000")
 
+    def test_get_loc_tz_aware(self):
+        # https://github.com/pandas-dev/pandas/issues/32140
+        dti = pd.date_range(
+            pd.Timestamp("2019-12-12 00:00:00", tz="US/Eastern"),
+            pd.Timestamp("2019-12-13 00:00:00", tz="US/Eastern"),
+            freq="5s",
+        )
+        key = pd.Timestamp("2019-12-12 10:19:25", tz="US/Eastern")
+        result = dti.get_loc(key, method="nearest")
+        assert result == 7433
+
```

Comment:
nitpick (not a blocker), can you put this a few tests up adjacent to the other tests that use the `method` kwarg

Reply:
sure. done.

Reply:
thanks